Tests/BaseObjectTests.swift

```diff
@@ -16,181 +19,184 @@ class BaseObjectTests: QuickSpec {
             }
 
             beforeEach {
-                baseObject = BaseObject()
+                fake = FakeBaseObject()
                 callbackWasCalled = false
             }
 
-            describe("on") {
-                it("Callback should be called on event trigger") {
-                    baseObject.on(eventName, callback: callback)
-                    baseObject.trigger(eventName)
-
-                    expect(callbackWasCalled) == true
+            context("initialization") {
+                it("Should not initialize EventDispatcher twice") {
```

Comment:
We agreed on not using `should`. You can rewrite this as: `it("doesn't initializes EventDispatcher twice")`